main.nf

```diff
@@ -810,10 +842,11 @@ process generate_cool{
  * STEP 6 - MultiQC
  */
 process multiqc {
+   label 'process_low'
    publishDir "${params.outdir}/MultiQC", mode: 'copy'
 
    when:
-   !params.skipMultiQC
+   !params.skip_multiqc
 
    input:
    file multiqc_config from ch_multiqc_config
```

Comment:
You're missing the new `ch_multiqc_custom_config` here that came in the template update.
```suggestion
   file multiqc_config from ch_multiqc_config
   file (mqc_custom_config) from ch_multiqc_custom_config.collect().ifEmpty([])
```

Reply:
done